index.json

```diff
@@ -19887,6 +20468,105 @@
     "name": "milestone",
     "description": "Activity related to milestones. The type of activity is specified in the `action` property of the payload object. For more information, see the \"[milestones](/en/rest/reference/issues#milestones)\" REST API.",
     "actions": ["closed", "created", "deleted", "edited", "opened"],
+    "properties": {
+      "milestone": {
+        "type": "object",
+        "description": "The milestone itself."
+      },
+      "changes": {
+        "type": "object",
+        "description": {
+          "0": "T",
+          "1": "h",
+          "2": "e",
+          "3": " ",
+          "4": "c",
+          "5": "h",
+          "6": "a",
+          "7": "n",
+          "8": "g",
+          "9": "e",
+          "10": "s",
+          "11": " ",
+          "12": "t",
+          "13": "o",
+          "14": " ",
+          "15": "t",
+          "16": "h",
+          "17": "e",
+          "18": " ",
+          "19": "m",
+          "20": "i",
+          "21": "l",
+          "22": "e",
+          "23": "s",
+          "24": "t",
+          "25": "o",
+          "26": "n",
+          "27": "e",
+          "28": " ",
+          "29": "i",
+          "30": "f",
+          "31": " ",
+          "32": "t",
+          "33": "h",
+          "34": "e",
+          "35": " ",
+          "36": "a",
+          "37": "c",
+          "38": "t",
+          "39": "i",
+          "40": "o",
+          "41": "n",
+          "42": " ",
+          "43": "w",
+          "44": "a",
+          "45": "s",
+          "46": " ",
+          "47": "`",
+          "48": "e",
+          "49": "d",
+          "50": "i",
+          "51": "t",
+          "52": "e",
+          "53": "d",
+          "54": "`",
+          "55": ".",
```

Comment:
This is an unintended side effect... How unfortunate that there's a property named `description`.

How should I handle this?